linux/pipenv_functions.bsh

```diff
@@ -62,5 +62,26 @@ function get_pipfile_hash()
 
 function get_pipfile_lock_hash()
 {
-  python -c "import json; f = json.load(open('$1', 'r')); print(f['_meta']['hash']['sha256'])"
-}
\ No newline at end of file
+  python -c "if True:
+    import json
+    f = json.load(open('$1', 'r'))
+    print(f['_meta']['hash']['sha256'])"
+}
+
+#**
+# .. function:: get_pipenv_virtualenv_location
+#
+# Print the virtualenv location for a given Pipefile
+#
+# :Arguments: ``$1`` - pipenv project directory (directory and Pipfile does not need to exist)
```

Comment:
pipenv project name (i.e., the directory in which the Pipfile is located; directory and Pipfile need not exist)

Reply:
The documentation does refer to this as "project’s root directory" and "project path", so saying "project name" seems less clear

Reply:
project root directory seems good.